### PR TITLE
Introducing ops.py - some functions to help with setting up the production environment. 

### PR DESCRIPTION
**Pull request by maxwellmb**
## Describe your changes

The PR introduces ops.py, which gives a simple functions laying out the functionality of the production environment. 

## Type of change
- New feature (non-breaking change which adds functionality)

## Checklist before requesting a review
- [x] I have linted my code
- [x] I have verified that all unit tests pass in a clean environment and added new unit tests, as needed
- [x] I have verified that all docstrings are properly formatted and added new documentation, as needed

### CODE REVIEW

File: corgidrp/ops.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+
+import corgidrp.caldb as caldb
+import corgidrp.walker as walker
+import argparse
+
+
+def step_1_initialize():
```
Comment (semaphoreP) [inline corgidrp/ops.py:7]: Let's add some docstrings to these three functions, especially since they are outwards facing.

**Review by semaphoreP — COMMENTED** [review]
Looks good. Just add some docstrings to the ops.py functions

**Comment by maxwellmb** [comment]
@semaphoreP Added docstrings! Should be good to go assuming the tests pass.

**Comment by semaphoreP** [comment]
sorry, I made Kevin change a keyword from `obstype` to `arrtype` which caused your test to fail.

**Comment by maxwellmb** [comment]
That's what I get for being lazy and not merging main and running my own test....

Ok, tests passing now.

**Review by semaphoreP — APPROVED** [review]

**Comment by semaphoreP** [comment]
No worries. I often do the same. That's why we have this infrastructure in place!

**Pull request merged**